### PR TITLE
registry: remove local network mode

### PR DESCRIPTION
**Pull request by yolile**
To be merged after/with:
- https://github.com/open-contracting/kingfisher-process/pull/347
- https://github.com/open-contracting/data-registry/pull/173
- https://github.com/open-contracting/pelican-frontend/pull/72
- https://github.com/open-contracting/deploy-pillar-private/pull/22

### CODE REVIEW

File: salt/docker_apps/files/kingfisher_process.yaml [hunk 1]
```diff
@@ -13,7 +14,9 @@ x-daemon: &daemon
 services:
   django:
     <<: *daemon
-    command: "./gunicorn_runner.sh {{ pillar.docker_apps.kingfisher_process.port }}"
+    # The container port must match the one exposed in the Dockerfile
```
Comment (jpmckinney) [inline salt/docker_apps/files/kingfisher_process.yaml:17]: This comment is obvious, no? (Here and elsewhere)

File: salt/docker_apps/files/pelican_frontend.yaml [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-  restart: unless-stopped
-
 services:
   django:
-    <<: *daemon
-    command: "./gunicorn_runner.sh {{ pillar.docker_apps.pelican_frontend.port }}"
+    image: "ghcr.io/open-contracting/pelican-frontend-django:latest"
+    restart: unless-stopped
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```
Comment (jpmckinney) [inline salt/docker_apps/files/pelican_frontend.yaml:8]: nitpick: In other files, restart is declared after network_mode/extra_hosts.

```suggestion
    extra_hosts:
      - "host.docker.internal:host-gateway"
    restart: unless-stopped
```

**Review by jpmckinney — COMMENTED** [review]
What are the docs that explain `host.docker.internal:host-gateway`? You can just add a comment in https://github.com/open-contracting/deploy/issues/309 with the link(s).

File: salt/docker_apps/files/kingfisher_process.yaml (hunk 1 above)
Comment (yolile) [reply salt/docker_apps/files/kingfisher_process.yaml]: Removed

File: salt/docker_apps/files/pelican_frontend.yaml (hunk 2 above)
Comment (yolile) [reply salt/docker_apps/files/pelican_frontend.yaml]: Reordered

**Review by jpmckinney — APPROVED** [review]
Looks good. Please just add a link to #309 to where to learn more about `host.docker.internal`, etc. networking.

**Pull request merged**